tests/data/state_estimation/current-sensor/basic-local-current-sensor/sym_output.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "1.0",
+  "type": "sym_output",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {
+        "id": 1,
+        "energized": 1,
+        "u_pu": 0.9999918620376325,
+        "u": 9999.918620376324,
+        "u_angle": 1.222133428418590e-13
+      },
+      {
+        "id": 2,
+        "energized": 1,
+        "u_pu": 0.9999926200230093,
+        "u": 9999.926200230093,
+        "u_angle": 1.204079620603813e-8
```

Comment:
why not just make it `0` by means of `rtol` and `atol`? or is this the exact solution? does this follow out of the bounds of `rtol` and `atol` or do we need to increase the reactance in the grid and/or reduce `rtol` and `atol` to prove that this works as intended?